tensorflow/contrib/makefile/Makefile

```diff
@@ -38,8 +38,9 @@ HOST_OBJDIR := $(MAKEFILE_DIR)/gen/host_obj/
 HOST_BINDIR := $(MAKEFILE_DIR)/gen/host_bin/
 HOST_GENDIR := $(MAKEFILE_DIR)/gen/host_obj/
 
-# Find the current Eigen version name from the Bazel build file
-EIGEN_HASH := $(shell cat eigen.BUILD | grep archive_dir | head -1 | cut -f3 -d- | cut -f1 -d\")
+# Find the current Eigen version name from the cmake configuration
+EIGEN_HASH := $(shell cat tensorflow/contrib/cmake/external/eigen.cmake | \
```

Comment:
Yeah, that's a good idea.
I started doing that, but found it a bit brittle because the `workspace.bzl` doesn't state "this is the eigen hash" in a clear way.

I could define an `eigen_hash` variable in `workspace.bzl` and add a comment that this line is parsed from the makefile and cmake builds.